src/rgw/rgw_rados.cc

```diff
@@ -3115,7 +3115,7 @@ class RGWDataNotifier : public RGWRadosThread {
   RGWDataNotifierManager notify_mgr;
 
   uint64_t interval_msec() override {
-    return cct->_conf->rgw_md_notify_interval_msec;
+    return cct->_conf->get_val<int>("rgw_data_notify_interval_msec");
```

Comment:
getting a `FAILED assert("wrong type or option does not exist" == nullptr)` from this line